src/test/java/atas/logic/commands/sessionlist/AddSessionCommandTest.java

```diff
@@ -241,6 +241,11 @@ public void setCurrentSessionTrue() {
         public boolean returnCurrentSessionEnabledStatus() {
             return false;
         }
+
+        @Override
+        public Index generateRandomStudentIndex() {
+            return null;
```

Comment:
Just a query, would returning null be a good idea?

Reply:
This implementation is under the ModelStub and this method is not used at all, so I thought returning null wouldn't make much of a difference either way.

Do you have any alternative suggestions for this part?